x-pack/platform/plugins/shared/integration_assistant/server/integration_builder/build_integration.ts

```diff
@@ -77,7 +77,7 @@ export async function buildPackage(integration: Integration): Promise<Buffer> {
   return zipBuffer;
 }
 export function isValidName(input: string): boolean {
-  const regex = /^[a-zA-Z_]+$/;
+  const regex = /^[a-zA-Z0-9_]+$/;
```

Comment:
Why do we have different regular expressions here and in https://github.com/elastic/kibana/blob/6bf5e68e06bf60df0075d6d60f57a836a1d49fc8/x-pack/platform/plugins/shared/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/data_stream_step.tsx#L46 ? Wouldn't it be better to extract it into the common file and input twice?

Reply:
Yeah makes sense.. Fixed in https://github.com/elastic/kibana/pull/205029